posthog/schema.py

```diff
@@ -1446,6 +1523,49 @@ class PropertyGroupFilterValue(BaseModel):
     ]
 
 
+class QueryCombinedResponse(BaseModel):
+    model_config = ConfigDict(
+        extra="forbid",
+    )
+    clickhouse: Optional[Union[Any, str, List[DatabaseSchemaQueryResponseField]]] = None
+    columns: Optional[Union[Any, List, List[DatabaseSchemaQueryResponseField]]] = None
+    error: Optional[Union[Any, str, List[DatabaseSchemaQueryResponseField]]] = None
+    errors: Optional[Union[Any, List[HogQLNotice], List[DatabaseSchemaQueryResponseField]]] = None
+    explain: Optional[Union[Any, List[str], List[DatabaseSchemaQueryResponseField]]] = None
+    hasMore: Optional[Union[Any, bool, List[DatabaseSchemaQueryResponseField]]] = None
+    hogql: Optional[Union[Any, str, List[DatabaseSchemaQueryResponseField]]] = None
+    inputExpr: Optional[Union[Any, str, List[DatabaseSchemaQueryResponseField]]] = None
+    inputSelect: Optional[Union[Any, str, List[DatabaseSchemaQueryResponseField]]] = None
+    isValid: Optional[Union[Any, bool, List[DatabaseSchemaQueryResponseField]]] = None
+    isValidView: Optional[Union[Any, bool, List[DatabaseSchemaQueryResponseField]]] = None
+    is_cached: Optional[Union[Any, bool, List[DatabaseSchemaQueryResponseField]]] = None
+    last_refresh: Optional[Union[Any, str, List[DatabaseSchemaQueryResponseField]]] = None
+    limit: Optional[Union[Any, int, List[DatabaseSchemaQueryResponseField]]] = None
+    missing_actors_count: Optional[Union[Any, int, List[DatabaseSchemaQueryResponseField]]] = None
+    modifiers: Optional[Union[Any, HogQLQueryModifiers, List[DatabaseSchemaQueryResponseField]]] = None
+    next: Optional[Union[str, Any, List[DatabaseSchemaQueryResponseField]]] = None
+    next_allowed_client_refresh: Optional[Union[Any, str, List[DatabaseSchemaQueryResponseField]]] = None
+    notices: Optional[Union[Any, List[HogQLNotice], List[DatabaseSchemaQueryResponseField]]] = None
+    offset: Optional[Union[Any, int, List[DatabaseSchemaQueryResponseField]]] = None
+    query: Optional[Union[Any, str, List[DatabaseSchemaQueryResponseField]]] = None
```

Comment:
These types look a bit odd. Should they all be an union with `Any` and `List[..Field]`?

Reply:
It does look odd, I can check.
Also... we probably need to improve this later on, since it's basically now "Oh you might get any of these fields. We can't tell. Have fun."

Reply:
The `Any` and `List[DatabaseSchemaQueryResponseField]` comes from types that define every field as being possibly that... so by combining, it ends up looking like this.

But I have an alternative where we say the response is "any of the responses of our types". Which looks a bit better.